src/reporters/json.ts

```diff
@@ -515,23 +518,25 @@ export default class JSONReporter extends BaseReporter {
     );
   }
 
-  async writeScreenshotBlocks(journey: Journey, step: Step, data: string) {
+  async writeScreenshotBlocks(journey: Journey, screenshot: Screenshot) {
     const { blob_mime, blocks, reference } = await getScreenshotBlocks(
-      Buffer.from(data, 'base64')
+      Buffer.from(screenshot.data, 'base64')
     );
     for (let i = 0; i < blocks.length; i++) {
       const block = blocks[i];
       this.writeJSON({
         type: 'screenshot/block',
+        timestamp: screenshot.timestamp,
         _id: block.id,
         blob: block.blob,
         blob_mime,
       });
     }
     this.writeJSON({
       type: 'step/screenshot_ref',
+      timestamp: screenshot.timestamp,
```

Comment:
We are using the document timestamp in the UI. I'm assuming we should move to using this timestamp.

Reply:
We rewrite it to document timestamp here - https://github.com/elastic/synthetics/blob/926e7d812be37f99e3af60bcc5d946bc3d9e2209/src/reporters/json.ts#L584